00-programming-fundamentals/exercises/merge-conflict-remote.md

```diff
@@ -0,0 +1,102 @@
+# Merge Conflict Carnival - Zoom Edition
+
+## Overview
+
+This year the Ada staff has generously agreed to contribute their culinary skills to a coding bootcamp booth offering food & info pamphlets at the [Georgetown Carnival](http://www.georgetowncarnival.com/). It's the day before the festivities kick off and _someone's_ Cavalier King Charles Spaniel has done his best to shred all of their recipes! Oh no! Can you help Ada staff put everything back together and save Carnival?
+
+In this activity you'll be working with your fellow Adies to put together multiple scraps from a single recipe using Git. You'll accomplish this by successively merging in content from each individuals' copy of the recipes repo. Once all the of merges are finished (and any merge conflicts resolved) you'll have a complete recipe document which can be submitted to the original repository with a pull request.
+
+## Learning Goals
+
+The goal of this activity is to give you exposure to the process of resolving merge conflicts. Because merge conflicts are a natural part of any collaborative development effort, this is an important skill to have as a developer.
+
+With this activity you will be able to encounter and resolve merge conflicts in a controlled scenario where the correct resolution of all merge conflicts is clear.
+
+## Instructions
+
+### Baseline Setup
+
+Before we can start putting the recipe together we need to make sure that everyone in your group is working from the same starting point and can effectively contribute to the recipe building effort.
+
+1. **Split into groups**: The class will be divided into groups using the breakout sections feature of Zoom, these assignments are random.
+   1. Breakout Room 1:  Aardvark
+   2. Breakout Room 2:  Tarsier
```

Comment:
I know some people have strong opinions about using 1s for all numbered lists. But I assume this was an intentional choice for some particular reason.